tcmur_cmd_handler.c

```diff
@@ -456,6 +456,134 @@ static int handle_read(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return async_handle_cmd(dev, cmd, read_work_fn);
 }
 
+/* format unit */
+struct format_unit_state
```

Comment:
We normally put the opening { on the same line as the struct's name like:

struct format_unit_state {